Customizing/global/plugins/Services/UIComponent/UserInterfaceHook/GEVCockpit/classes/class.ilGEVCockpitUIHookGUI.php

```diff
@@ -199,9 +198,27 @@ protected function getSubMenuHTML($current_skin) {
 			$tpl->parseCurrentBlock();
 			$count++;
 		}
+
+		$tpl->addCss("./Customizing/global/plugins/Services/UIComponent/UserInterfaceHook/GEVCockpit/templates/jquery.bxslider.css");
+		$js_tpl = new ilTemplate("./Customizing/global/plugins/Services/UIComponent/UserInterfaceHook/GEVCockpit/templates/tpl.submenu_slider.js.html", false, false);
```

Comment:
Das ist das Stückchen Javascript, in dem auch die Einstelungen für das bx-slider Plugin gemacht werden. Ist wäre kein Problem das in die tpl.submenu.html zu überführen.